aioshelly/const.py

```diff
@@ -144,10 +144,13 @@
 # Firmware 1.0.99 release date
 GEN3_MIN_FIRMWARE_DATE = 20231102
 
+# Firmware 1.5.x release date
+# Temporary use beta release to allow BluTrv support
+GEN3_GATEWAY_MIN_FIRMWARE_DATE = 20241219
```

Comment:
Firmware `20250109-090619/1.5.0-beta1-g16e0ef4` is available.
```suggestion
GEN3_GATEWAY_MIN_FIRMWARE_DATE = 20250109
```